src/python/pants/init/engine_initializer.py

```diff
@@ -87,7 +87,7 @@ def goal_consumed_subsystem_scopes(self, goal_name: str) -> Tuple[str, ...]:
             return tuple()
         consumed_types = self.goal_consumed_types(goal_product)
         return tuple(
-            sorted({typ.options_scope for typ in consumed_types if issubclass(typ, Subsystem)})  # type: ignore[misc]
+            sorted({typ.options_scope for typ in consumed_types if issubclass(typ, Subsystem)})
```

Comment:
Ha. That reminds me I wanted to make this change because it was so confusing to add MyPy to our options code